src/lib/components/forms/form/form-entry-list/terra-form-entry-list.component.ts

```diff
@@ -126,6 +126,7 @@ export class TerraFormEntryListComponent implements OnInit, OnChanges, ControlVa
             this.formArray.push(
                 TerraFormHelper.createNewControl(this.inputFormField.defaultValue, this.inputFormField)
             );
+            this.inputFormGroup.markAsDirty();
```

Comment:
Ich glaube du solltest eher das formArray as dirty markieren. Das wird sich zwar wahrscheinlich auch auf den parent (die `inputFormGroup`) auswirken, aber ich denke das wäre das richtige control.

Reply:
Wenn das Array `dirty`sein soll, dann sollte auch das Array als solches markiert werden. Status wie dirty, valid usw. werden in einer FormGroup immer nach oben gebubbelt 🙂

Reply:
formArray funktioniert 👍